bin/locales

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env node
+/*
+ * This Source Code is subject to the terms of the Mozilla Public License
+ * version 2.0 (the 'License'). You can obtain a copy of the License at
+ * http://mozilla.org/MPL/2.0/.
+ */
+
+/* eslint-disable no-console */
+
+const fs = require('fs');
+const glob = require('glob');
+
+glob('locales/*/*.properties', function (err, files) {
+  if (err) return console.error('Error finding files', err);
+  files.forEach(function(path) {
+    fs.createReadStream(path).pipe(fs.createWriteStream(`locale/${parseLang(path)}.properties`));
+  });
+});
+
+function parseLang(path) {
+  const segments = path.split('/');
```

Comment:
Not that we probably care right now, but I don't think this will work on Windows.

We could potentially use the `path` module and use a combo of `path.dirname()` and `path.sep`, a la:

```js
function parseLang(file) {
  return path.dirname(file)
    .split(path.sep)
    .pop();
}